terminal-here.el

```diff
@@ -41,18 +40,13 @@
     (list "cmd.exe" "/C" "start" "cmd.exe"))
 
    ;; Probably X11!
-   (t '("x-terminal-emulator"))))
-
-
-(defcustom terminal-here-terminal-command
-  #'terminal-here-default-terminal-command
+   (t '("x-terminal-emulator")))
   "The command used to start a terminal.
 
 Either a list of strings: (terminal-binary arg1 arg2 ...); or a
 function taking a directory and returning such a list."
   :group 'terminal-here
-  :type '(choice (repeat string)
-                 (function)))
+  :type '(choice (repeat string)))
```

Comment:
The option to set this as a function is needed because some terminals don't automatically open to the current directory but instead need to be passed a command line argument telling them which directory to open.

Reply:
OK, then I'll close the pull request.